scripts/Iris.json

```diff
@@ -0,0 +1,88 @@
+{
+    "citation": "R. A. Fisher. 1936. The Use of Multiple Measurements in Taxonomic Problems. and Asuncion, A. & Newman, D.J. (2007). UCI Machine Learning Repository [http://www.ics.uci.edu/~mlearn/MLRepository.html]. Irvine, CA: University of California, School of Information and Computer Science.
+
+",
```

Comment:
Looks like there are a couple of newlines that shouldn't be here. They prevent this from working for me (it doesn't show up in `retriever ls`).